README.md

```diff
@@ -185,7 +185,9 @@ Want to see a more advanced example? [Checkout our test micro kernel](Tests/Fixt
 dunglas_action:
     autodiscover:         # Autodiscover action classes stored in the configured directory of bundles and register them as service.
         enabled:   true
-        directory: Action # The directory name to autodiscover in bundles.
+        directories: # The directories name to autodiscover in bundles.
+            action: [ Action ]
+            command: [ Command ]
```

Comment:
Is it safe to keep the same directory name than for "old school" commands? If a project register a bundle containing "old school" commands, will it work safely?